airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/NoOpJdbcDestinationHandler.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.integrations.destination.jdbc.typing_deduping
+
+import com.fasterxml.jackson.databind.JsonNode
+import io.airbyte.cdk.db.jdbc.JdbcDatabase
+import io.airbyte.integrations.base.destination.typing_deduping.AirbyteType
+import io.airbyte.integrations.base.destination.typing_deduping.DestinationInitialStatus
+import io.airbyte.integrations.base.destination.typing_deduping.Sql
+import io.airbyte.integrations.base.destination.typing_deduping.StreamConfig
+import io.airbyte.integrations.base.destination.typing_deduping.StreamId
+import org.jooq.SQLDialect
+
+class NoOpJdbcDestinationHandler<DestinationState>(
+    databaseName: String,
```

Comment:
is there even a point to taking params / extending jdbcdestinationhandler? could this just be `NoopDestinationHandler: DestinationHandler<MinimumDestinationState>`?

Reply:
That does seem like the right path but the [AbstractJdbcDestination]([url](https://github.com/airbytehq/airbyte/blob/7c98ebe15efcc003b9bc977cff075e312dc49f0f/airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/AbstractJdbcDestination.kt#L169)) currently requires it